sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/BullseyeParser.java

```diff
@@ -57,8 +57,6 @@ public BullseyeParser(final String baseDir) {
   @Override
   public void processReport(final Project project, final SensorContext context, File report, final Map<String, CoverageMeasuresBuilder> coverageData)
     throws XMLStreamException {
-    CxxUtils.LOG.info("Parsing 'Bullseye' format");
```

Comment:
There are different coverage parsers. This is to know which coverage parser is in use. But could be debug?